package.json

```diff
@@ -1,62 +1,24 @@
 {
   "name": "querystring",
   "id": "querystring",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Let's not bump version in this PR

Reply:
I have changed it back to `v0.2.0` but current version of package currently ships [some wierd binary files](https://unpkg.com/browse/querystring@0.2.0/), so giving minor bump might be a good thing.

I know this will be tree shaken most off time but this will save 15-20 kb on package download

Reply:
I agree.

Whole point is to handle version release outside of this PR (I believe versioning should not be mixed in features/fixes PR's)